schedule_lessons/dashboard/views.py

```diff
@@ -895,6 +895,21 @@ def save_tutorial_preferences(request):
     user_profile.save()
     return HttpResponse(status=200)
 
+@login_required
+def settings(request):
+    notifications = Notification.objects.filter(
+        user=request.user).order_by("-created_on")
+    context = {
+        "notifications": []
```

Comment:
question: how come context doesn't have a property called `unread_notifications` even tho it is being used later on?

Reply:
True actually, I opened up issue #139 about this so we can clean this up later since there are other instances where I do this in this file.